text/0172-health-api.md

```diff
@@ -0,0 +1,53 @@
+# Health API
+
+## 1. Summary
+
+This specification describes the health API endpoint. The `/health` route allows to verify the status and availability of a Meilisearch instance.
+
+## 2. Motivation
+
+To know the status of a Meilisearch instance, `/health` is a public route that can be long pooled with simple monitoring tools.
+
+## 3. Functional Specification
+
+### 3.1. `health` API resource properties
+
+| Field                                            | Type            | Required |
+|--------------------------------------------------|-----------------|----------|
+| [status](#311-status)                            | String          | True     |
```

Comment:
`Required`?
It's not a parameter, you don't have a choice, we should remove this last column right?

Reply:
The boolean required is also mentioned in the response definition of resources because we sometimes have conditional response fields (for example in the search).